### ISSUE TITLE
error when installation 

### ISSUE THREAD

**Issue by A00001111** [kaltura/moodle_plugin#438]
when install this plugin this error shown for me

```
Install plugin from ZIP file
The plugin ZIP package must contain just one directory, named to match the plugin name. The file provided is not a valid plugin ZIP package.
```

and when make folder with its name and add folders to it also notworking

**Comment by ctam** [comment]
I do not think you can install this plugin through the web interface. This repo is actually a bundle of multiple plugins and the only why I can get this to work is to unzip them into the Moodle's root directory at the command line.
